acpi.adoc

```diff
@@ -33,7 +33,7 @@ available to an OS loader via the standard `UEFI EFI_GRAPHICS_OUTPUT_PROTOCOL` i
  * Revision 4 or later of SPCR.
  * For NS16550-compatible UARTs:
  ** Use `Interface Type` 0x12 (16550-compatible with parameters defined in Generic Address Structure).
- ** There MUST be a matching AML device object with compatible ID `RSCV0003`.
+ ** There MUST be a matching AML device object with `_HID` (Hardware ID) or `_CID` (Compatibility ID) `RSCV0003`.
```

Comment:
Thanks for adding HID as well. ACPI spec (6.1.2) refers as Compatible ID itself. So, better to have "Compatible ID" instead of "Compatibility ID" ?